kernelci.org/content/docs/team/tsc.md

```diff
@@ -0,0 +1,267 @@
+---
+title: "Technical Steering Committee"
+date: 2021-02-10T11:41:14Z
+draft: true
+description: "Core developers and maintainers"
+---
+
+The Technical Steering Committee (TSC) is a team of people who are responsible
+for keeping the KernelCI project in a good shape and driving its development.
+
+The rules for adding or removing members from the TSC are defined in the LF
+project charter.  Typically, major contributors eventually become members and
+those who have stopped contributing for an extended period of time may be
+removed.  This is done in agreement with the TSC and the AB.
+
+As of today, the committee is composed of the following members:
+
+* [Kevin Hilman](mailto:<khilman@baylibre.com>) - `khilman`
+* [Mark Brown](mailto:<broonie@kernel.org>) - `broonie`
+* [Guillaume Tucker](mailto:<guillaume.tucker@collabora.com>) - `gtucker`
+* [Corentin Labbe](mailto:<clabbe@baylibre.com>) - `montjoie`
+* [Nikolai Kondrashov](mailto:<Nikolai.Kondrashov@redhat.com>) - `spbnick`
```

Comment:
Can you put my personal e-mail here: spbnick@gmail.com instead?

Reply:
Done.